ojd_daps_skills/pipeline/extract_skills/extract_skills.py

```diff
@@ -365,22 +384,30 @@ def map_skills(self, predicted_skills):
 
         return job_skills_matched_formatted
 
-    def extract_skills(self, job_adverts, map_to_tax=True):
+    def extract_skills(
+        self, job_adverts_skills: Union[str, List[dict]], format_skills=False
```

Comment:
I feel funny about this. It does feel odd that you would use a function called "extract_skills" on a list of skills..

Reply:
deal